comfyui_custom_nodes/webui_save_image.py

```diff
@@ -31,14 +31,14 @@ def save_image(self, location, images):
         for image in images:
             pil_image = to_pil_image(image.permute(2, 0, 1))
             save_path = os.path.join(data_path, output_dir)
-            filename, _ = WebuiSaveImage.webui_save_image(image=pil_image, path=save_path, basename='')

```

Comment:
refering to this

Reply:
it's not really important because we don't need the returned values, but I wonder why ipc would return `None`.

Reply:
ah nvm I found out why